pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -188,12 +188,13 @@ protected void removeSegmentFromStore(String tableNameWithType, String segmentId
       try {
         if (pinotFS.exists(fileToMoveURI)) {
           // Overwrites the file if it already exists in the target directory.
-          pinotFS.move(fileToMoveURI, deletedSegmentDestURI, true);
-          // Updates last modified.
-          // Touch is needed here so that removeAgedDeletedSegments() works correctly.
-          pinotFS.touch(deletedSegmentDestURI);
-          LOGGER.info("Moved segment {} from {} to {}", segmentId, fileToMoveURI.toString(),
-              deletedSegmentDestURI.toString());
+          if (pinotFS.move(fileToMoveURI, deletedSegmentDestURI, true)) {
```

Comment:
Should an message be logged if move returns false? If we can rely on move to always log, then you can remove the additional log in PinotLLCRealtimeSegmentManager.java

Reply:
I think it's essential to log when returning false because the false result may be returned from the deep inner logic of different FS.